blog/test.py

```diff
@@ -20,3 +22,16 @@ def test_content_cleaned_html(self):
         article = Article(title='',
                           content='<p><em>emphased content</em><i>also emphased</i></p>').save()
         self.assertEqual(article.content, '<p><em>emphased content</em><em>also emphased</em></p>')
+
+    def test_delete_article_and_folders_pictures(self):
```

Comment:
Just make it simple:

1. create an article [like we do here](https://github.com/neomad-team/neomad.org/blob/master/blog/test.py#L8)
2. check the folder exists
3. delete the article
4. ensure the folder is deleted

All in all, it would look like:
```
article = Article(title='', content='<p>content</p>').save()
self.assertTrue(os.path.isdir(article.get_images_path())
article.delete()
self.assertFalse(os.path.isdir(article.get_images_path())
```